lib/rules/property-no-unknown/__tests__/index.js

```diff
@@ -34,6 +34,12 @@ testRule(rule, {
     {
       code: ".foo { *width: 100px; }",
       description: "ignore CSS hacks"
+    },
```

Comment:
Can we please pull this out into its own `testRule` block with `syntax: styled`, please?

We should place it at the end of the file.